sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -1230,10 +1230,11 @@ def run_Read(self, transform_node, options):
       if transform.source.id_label:
         step.add_property(
             PropertyNames.PUBSUB_ID_LABEL, transform.source.id_label)
-      if transform.source.with_attributes:
-        # Setting this property signals Dataflow runner to return full
-        # PubsubMessages instead of just the data part of the payload.
-        step.add_property(PropertyNames.PUBSUB_SERIALIZED_ATTRIBUTES_FN, '')
+
+      # Setting this property signals Dataflow runner to return full
+      # PubsubMessages instead of just the data part of the payload.
+      step.add_property(PropertyNames.PUBSUB_SERIALIZED_ATTRIBUTES_FN, '')
```

Comment:
Do you know if there's any significant perf hit due to pulling in full messages instead of just data ?

Reply:
Hi Daniel, do you notice any performance variance between reading from pubsub with `with_attributes=True` and `with_attributes=False` when running on streaming service? @dpmills

Reply:
Is there any benchmark that we can run to make sure there's no significant perf hit ?
I'm +1 for this simplification if there's no perf hit.

Reply:
Hi Cham, I'm curious what kind of performance metrics we are talking about here since it's the streaming case.

Reply:
Looks like internal streaming benchmarks (for example, PythonStreamingWordCount) track throughput and system lag.

@slavachernyak might have additional info here.